config/moment.js

```diff
@@ -0,0 +1,8 @@
+const moment = require('moment-timezone');
+const { TIME_ZONE } = require('./constants');
+
+moment.tz(TIME_ZONE);
+
+module.exports = {
+  moment
+};
```

Comment:
As I was telling you, it seems to me that using moment only to give time does not bring much benefit.